spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/util/RetryUtil.scala

```diff
@@ -0,0 +1,21 @@
+package com.datastax.spark.connector.util
+
+import scala.annotation.tailrec
+import scala.util.{Failure, Success, Try}
+
+object RetryUtil {
```

Comment:
yep :) I thought I would need something like this in live code, but you are right we don't need this currently.